pipeline/qc.rules

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) 2018. Mount Sinai School of Medicine
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+This file contains QC-related processing rules.
+"""
+
+import glob
+
+def _get_all_fastq_files(_):
+    return glob.glob("%s/*.fastq.gz" % WORKDIR)
```

Comment:
Just sanity checking -- are the files guaranteed to have this extension and none of {.fasta, .fa, .fa.gz, .fq, .fq.gz}?

Reply:
The files we're working with so far only have this extension. Also, the .fastq.gz assumption is built into a few other places in the code - probably worth fixing at some point, but not as part of this